### ISSUE TITLE
[Error] clang-cl 12 is not supported !!

### ISSUE THREAD

**Issue by redradist** [conan-io/cmake-conan#332]
I got such error:
```
-- Conan executing: C:/Python/CPython39/Scripts/conan.exe install . --remote conan-center --build missing --settings build_type=Debug --settings compiler=clang --settings compiler.version=12 --settings compiler.libcxx=libc++
ERROR: Invalid setting '12' is not a valid 'settings.compiler.version' value.
Possible values are ['3.3', '3.4', '3.5', '3.6', '3.7', '3.8', '3.9', '4.0', '5.0', '6.0', '7.0', '7.1', '8', '9', '10']
```

Maybe it is not needed to add new version to array each time ?
Maybe we just need some version like '*' ? I think it is better to discuss in conan, but I also mentioned here

**Comment by czoido** [comment]
Hi @redradist,
clang 12 is not in the _settings.yml_ distributed with Conan because it has [not been officially released yet](https://releases.llvm.org/), but you can customize your _settings.yml_ and add that version if you need to use that. Please check the [documentation on how to customize settings](https://docs.conan.io/en/latest/extending/custom_settings.html#custom-settings).
Please tell me if this helps or if you have any further questions.

**Comment by redradist** [comment]
@czoido 
> 
> 
> Hi @redradist,
> clang 12 is not in the _settings.yml_ distributed with Conan because it has [not been officially released yet](https://releases.llvm.org/), but you can customize your _settings.yml_ and add that version if you need to use that. Please check the [documentation on how to customize settings](https://docs.conan.io/en/latest/extending/custom_settings.html#custom-settings).
> Please tell me if this helps or if you have any further questions.

Yeah, I know ... but does it make sense always ignore not released versions ?
Maybe better to add clang 12 to unstable versions ?

I mean the following to create in `settings.yml` unstable versions and allow to use such compiler with flag `--unstable` ?

**Comment by czoido** [comment]
I'm transferring this issue to Conan repo as this is not directly related with cmake-conan but with Conan tool itself.
